nightwatch.conf.js

```diff
@@ -1,85 +1,17 @@
 /* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "chromedriver" }] */
+const nightwatchConfig = require('terra-toolkit/lib/nightwatch/nightwatch.config.js').default;
+const webpackConfig = require('./packages/terra-consumer-site/webpack.config.js');
+const getPackageTestDirectories = require('terra-toolkit/lib/nightwatch/setup-helper.js').getPackageTestDirectories;
 
-const fs = require('fs');
-const globSync = require('glob').sync;
-const packages = require('./lerna.json').packages;
-const chromedriver = require('chromedriver');
+let srcFolders;
+const isRepoTest = !process.cwd().includes('/packages/');
```

Comment:
Using the following might be better if there is concern for windows development:

```
const isRepoTest = (__dirname === process.cwd());

if (isRepoTest) {
  specs = getPackageTestDirectories('lerna.json');
} else {
  srcFolders =  path.join('tests', 'nightwatch');
}
```